server/sonar-web/src/main/js/components/SelectList/SelectListListContainer.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import SelectListListElement, { SelectListItem } from './SelectListListElement';
+
+interface Props {
+  elements: SelectListItem[];
+  filter: string;
+  onSelect: (key: string | number) => void;
+  onUnselect: (key: string | number) => void;
+}
+
+export default class SelectListListContainer extends React.PureComponent<Props, {}> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This is the default value for this type parameter, so it can be omitted. ([typescript:S4157](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4157))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3152&issues=AWIkUgxm7DpOGbPQ2tAA&open=AWIkUgxm7DpOGbPQ2tAA)

[ISSUE_KEY]: AWIkUgxm7DpOGbPQ2tAA